README.md

```diff
@@ -252,4 +252,20 @@
 
 [Installation and Setup](http://sensu-plugins.io/docs/installation_instructions.html)
 
-Note:  In addition to the standard installation requirements the installation of this gem will require compiling the nokogiri gem.  Due to this you'll need certain developmemnt packages on your system.  On Ubuntu systems install build-essential, libxml2-dev and zlib1g-dev.  On CentOS install gcc and zlib-devel.
+Note:  In addition to the standard installation requirements the installation of this gem will require compiling the nokogiri gem.  Due to this you'll need certain development packages on your system.  On Ubuntu systems install build-essential, libxml2-dev and zlib1g-dev.  On CentOS install gcc and zlib-devel.
+
+## Authentication
+
+AWS credentials are required to execute these checks. Starting with AWS-SDK v2 there are several
+methods of passing credentials to the check:
+
+1. Set `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables
```

Comment:
I think we should include that there are multiple options here as well as an example or two as I have had to explain how to do this in the context of sensu. I mean we don't want to encourage people just exporting it the exact same way as part of the check definition...it should probably be sourced either within the check from a file or overall by the sensu process being started. They both have their advantages but sourcing on the check limits the scope of credentials needed and who gets it.